library/L0_Platform/stm32f10x/system_stm32f10x.h

```diff
@@ -0,0 +1,104 @@
+/**

```

Comment:
This file is useless and is only here for supporting the CMSIS platform, which we are an alternative to. So no need to keep this here. Delete this and remove it from the other files that include it.